### ISSUE TITLE
Issue about train.sh

### ISSUE THREAD

**Issue by hym97** [eth-sri/learch#1]
Hi:
I am trying to use some new models to explore paths for symbolic execution. However, there may exist some bugs in the training data preparation stage. 

Specifically, when I generate supervised training set via `train.sh`, the majority of the `reward` column is 0. I guess it is not normal behavior, and there may be some minor bugs within the source code. Could you help me to figure it out? Thank you very much.

Here is the code I use to generate training data:
![image](https://user-images.githubusercontent.com/78656286/161151564-03e013a7-5e00-4421-9a27-6b3dd8589eef.png)

Here is the link to the data produced (b2sum, base32, chcon): https://drive.google.com/drive/folders/1UvnNy69O81Jj4JXLBOAgj1M8q5eoiis0

**Comment by LostBenjamin** [comment]
Hi,

Thanks for your interest in learch! It is not surprising that some of the reward columns are 0. This is because most of generated test cases do not improve coverage. I don't see any files in your google drive folder and thus cannot analyze further.

Best,
Jingxuan

**Comment by hym97** [comment]
Thanks for a timely response. I have read through the code and solved the problem. I have an extra small question: when training LEARCH, how long do you run KLEE for each program to produce a supervised dataset (e.g. 60 secs)?  Thank you very much!

Cheers,
Yimeng

**Comment by hym97** [comment]
Thank you. That helps a lot!
